.CurrentChangelog.md

```diff
@@ -194,13 +202,28 @@ CI:
 * ci: reduce the name for a better log on workflows
 * ci(IWYU): add patch to avoid fedora failure (#4146)
 * ci(runner): use github hosted arm runner instead of ours
+* ci(StepSecurity): apply the restrict token permissions (#4192)
+* ci(StepSecurity): add step-security/harden-runner (#4190)
+* ci(StepSecurity): apply pinned dependencies (#4187)
+* ci(dependabot): bump github/codeql-action from 3.29.0 to 3.29.2 (#4250)
+* ci: Add support for llvm19 on macos brew (#4013)
+* ci: apply GITHUB TOKEN permission best practice
+* ci: pinned to specific actions' SHA (#4229)
+* ci(dependabot): bump actions/checkout from 3.6.0 to 4.2.2 (#4202)
+* ci(dependabot): bump step-security/harden-runner from 2.12.1 to 2.12.2 (#4218)
+* ci(dependabot): bump crazy-max/ghaction-chocolatey from 3.3.0 to 3.4.0 (#4200)
+* ci(dependabot): bump cachix/install-nix-action from 31.4.0 to 31.4.1 (#4219)
+* ci(commitlint): set body-max-line-length to warning level (#4222)
+* ci(dependabot): bump docker/setup-buildx-action from 3.10.0 to 3.11.1 in the docker group (#4203)
+* ci(sbom): migrate SBOM generation to REST API (#4196)
 * [Linux] Deprecate all CI workflows related to the manylinux2014 image
 * [nix] flake update and use LLVM 18 (#3839)
 * [macOS] The minimum macOS version is set to macos-13
 * [Misc] Fix build failure in Windows Server 2022 CI (#3899)
+* feat: unified all installer CI workflows (#4201)
 
 Thank all the contributors who made this release possible!
 
-abdelkoddous LHAJOUJI, alabulei1, dependabot[bot], Deveshi Dwivedi, dm4, Dmytrol, elhewaty, Fahd Ashour, fancybody, Gianpaolo Macario, grorge, Han-Wen Tsao, hydai, Jacob, junxiangMu, Karan, khongtrunght, kilavvy, LFsWang, Lîm Tsú-thuàn, Maxim Evtush, Michael Yuan, Oleg, omahs, operagxsasha, PeterD1524, Ruslan Tushov, Shen-Ta Hsieh, sridamul, Sridhar Sivakumar, Sylveon, Tenderyi, varunrmallya, vincent, Vladimir Cherkasov, Yi Huang, Yi-Ying He
+abdelkoddous LHAJOUJI, alabulei1, dependabot[bot], Deveshi Dwivedi, dm4, Dmytrol, elhewaty, Fahd Ashour, fancybody, Gianpaolo Macario, grorge, Han-Wen Tsao, hydai, Jacob, junxiangMu, Karan, Karan Lokchandani, khongtrunght, kilavvy, LFsWang, Lîm Tsú-thuàn, Maxim Evtush, Michael Yuan, Oleg, omahs, operagxsasha, PeterD1524, Ruslan Tushov, Shen-Ta Hsieh, Shen-Ta Hsieh(BestSteve), sridamul, Sridhar Sivakumar, StepSecurity Bot, Sylveon, Tenderyi, varunrmallya, vincent, Vladimir Cherkasov, Wang-Yang,  Li, Yi, Yi Huang, Yi-Ying He, YiYing He
```

Comment:
I think we should deleted the duplicated names which are automatically generated.

Reply:
I don't think we have to delete these duplicated names. It's nonsense if the contributors use different names, as we would then need to isolate them and remove the duplicate items.